puppet.go

```diff
@@ -0,0 +1,89 @@
+// Mgmt
+// Copyright (C) 2013-2016+ James Shubin and the project contributors
+// Written by James Shubin <james@shubin.ca> and the project contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"bufio"
+	"io"
+	"log"
+	"os/exec"
+	"strings"
+)
+
+func PuppetConfig(puppetParameter, puppetConf string) *GraphConfig {
```

Comment:
I considered plain `puppet`, but thought it would read quite ambiguous (what "puppet"? Is this an interface to an actual Puppet (REST?), is it a boolean flag...?)

Will think a little harder. Perhaps is fine and I'm just freaking out ;)